src/test/Drips.t.sol

```diff
@@ -5,7 +5,23 @@ import {DSTest} from "ds-test/test.sol";
 import {Hevm} from "./Hevm.t.sol";
 import {Drips, DripsReceiver} from "../Drips.sol";
 
-contract DripsTest is DSTest {
+contract PseudoRandomUtils {
+    bytes32 private constant SALT_NOT_SET = "salt not set";
+    bytes32 private salt = SALT_NOT_SET;
```

Comment:
Wise move with encapsulation of the randomness logic! In that case we don't even need the `SALT_NOT_SET` hack in hopes that the salt will never be genuinely set to `salt not set`, we can have a boolean flag for that.